db/schema.rb

```diff
@@ -65,6 +65,14 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "news_posts", force: :cascade do |t|
+    t.string "title"
+    t.date "occurred_at"
```

Comment:
what is this for?

Reply:
Ther is a date attached to every News. 
![image](https://user-images.githubusercontent.com/46072181/79079210-c13e7780-7cdb-11ea-9075-a0cc08f2e53e.png)

Reply:
I completely forgot that that can go in the description as markdown text field

Reply:
@amanpatel123 we could just use the created_at date